### DISCUSSION TITLE
stop watching etcd loop due to closed etcd event channel

### DISCUSSION THREAD

**Discussion by zhenzi0322** [milvus-io/milvus#23509] [Ideas & Feature requests]
[milvus.log](https://github.com/milvus-io/milvus/files/11263049/milvus.log)

**Comment by xiaofan-luan** [comment]
I thought you cluster is by k8s due to some reason.
Maybe check the k8s log and see what is going on?

**Reply by zhenzi0322** [reply]
https://github.com/milvus-io/milvus/files/11263049/milvus.log

**Comment by zhenzi0322** [comment]
我是独立版本部署的。

**Reply by xiaofan-luan** [reply]
> 我是独立版本部署的。

If you take a the log
"Root Coord disconnected from etcd, process will exit"
this means milvus disconnect with etcd.
1) check ETCD and see if can be connected.
2) might be other reasons why this pod has been killed, maybe docker logs [container-id] can help.
